### ISSUE TITLE
Emacs Flickering while scrolling

### ISSUE THREAD

**Issue by medwatt** [snowie2000/mactype#671]
I've been using MacType for a while and really miss it when it's not enabled for a particular program. I've been getting a lot of flickering and didn't know what was causing it. I came across a post on reddit where someone was experiencing the same behaviour. Disabling MacType makes the flickering go away.

Here's a short video I recorded to demonstrate what exactly is happening. The first part is with MacType disabled: https://streamable.com/dspk50

While the rendering with the MacType disabled might look OK, it's actually worse in reality. The screen recording software is making it look better than it is.

Is there some solution to this problem because I would really hate to disable MacType for Emacs.

**Comment by snowie2000** [comment]
Just turning off clipboxfix for emacs should be enough to solve the problem

**Comment by medwatt** [comment]
> Just turning off clipboxfix for emacs should be enough to solve the problem

Thanks for replying. This only seems to reduce the flickering somewhat; it does not remove it completely. Does it matter that I'm using MacType in registry mode ?

**Comment by snowie2000** [comment]
No. Emacs was reported as flickering before if I remembered it correctly.
Maybe you could try to enable hinting for it?
